corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -170,12 +203,16 @@ def _get_files_for_migration(self, files, file_name, do_re_check):
     def get_templates_for_migration(self, app_name, template_name, do_re_check):
         app_templates = get_all_template_paths_for_app(app_name)
         available_templates = self._get_files_for_migration(app_templates, template_name, do_re_check)
+        if template_name:
+            return available_templates
```

Comment:
if the app is already completed, it will (on purpose) still try to re-migrate that file if it's specified. This is by design, as doing it the other way around (originally) actually caused confusing for folks